bot/seasons/christmas/adventofcode.py

```diff
@@ -446,8 +542,18 @@ def member_from_json(cls, injson: dict) -> "AocMember":
             starboard=cls._starboard_from_json(injson["completion_day_level"]),
             local_score=injson["local_score"],
             global_score=injson["global_score"],
+            full_dict = injson
         )
 
+    @staticmethod
+    def _ts_from_int(unix: int) -> str:
+        """
+        Generates a String of the timestamp from a given Unix time
+        (Subtracts 18000 due to EST being UTC-5)
+        """
+        date = datetime.fromtimestamp(unix-18000)
```

Comment:
Why would you make times EST rather than UTC? Not everyone participating lives in eastern United States.